playground/frontend/lib/modules/examples/models/example_model.dart

```diff
@@ -65,6 +65,13 @@ class ExampleModel with Comparable<ExampleModel> {
     this.outputs = outputs;
   }
 
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) || other is ExampleModel && path == other.path;
```

Comment:
Please add brackets around `other is ExampleModel && path == other.path`, otherwise the path check will always be done.

Reply:
Hello, @KonradJanica! Thank you for the review! Fixed this issue.